package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@primer/react",
-  "version": "35.26.0",
+  "version": "35.26.1",
```

Comment:
v36.26.1 just published this morning so it is expected to see this difference. Thanks for the update